src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+header {
+  position: relative;
+  display: flex;
+  align-items: center;
+
+}
+
+.logo {
+  margin-left: 50px;
+  margin-top: 10px;
+}
+
+.nav {
+  position: absolute;
+  right: 50px;
+}
+
+.menu {
+
+  display: flex;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.menu li {
+  margin-left: 20px;
+}
```

Comment:
Don't use tag names for styling (except html and body)
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/cd71af1b-809d-4c24-a4a1-3646408edc0d)